sparqlQueries/template12.sparql

```diff
@@ -0,0 +1,27 @@
+PREFIX dct: <http://purl.org/dc/terms/>
+PREFIX dummy: <http://rdf.biosemantics.org/ontologies/dummy/>
+PREFIX rdcMeta: <http://rdf.biosemantics.org/ontologies/rd-connect/>
+PREFIX iao: <http://purl.obolibrary.org/obo/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX dcat: <http://www.w3.org/ns/dcat#>
+PREFIX go: <http://www.geneontology.org/formats/oboInOwl#>
+PREFIX obohp: <http://purl.obolibrary.org/obo/hp#>
+
+SELECT (COUNT( DISTINCT ?donorURI) as ?numberOfPersons)  ?phenotypeURI ?phenotype ?diseaseURI ?disease ?registry ?registryURI  {
+      
+ VALUES(?phenotypeInput) {("#plainText#")}   
+  
+    ?phenotypeURI  rdfs:label ?phenotype;
+     go:inSubset obohp:hposlim_core.  
+	FILTER regex(str(?phenotype), ?phenotypeInput, "i") .
```

Comment:
This line contains a tab for indentation (while the rest of the file seems to use spaces).

Reply:
I will fix it